base/recipes/gearman.rb

```diff
@@ -73,10 +73,10 @@
   owner "root"
   group "root"
   mode "0755"
-  notifies :run, "execute[gearman systemcl daemon-reload]", :immediately
+  notifies :run, "execute[gearman systemctl daemon-reload]", :immediately unless node[:container]
```

Comment:
@Bellardia I never had success with this though, it looks like my syntax may have been off, ever-so-slightly.